gem/exe/ruby-prettier

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+def prettier
+  target = if RUBY_PLATFORM =~ /darwin/
```

Comment:
`pkg` seems to support `platform freebsd, linux, alpine, macos, win`, but defaults to macos, linux, windows.

Reply:
The defaults you chose make a lot of sense. Let's go with those until we get a report for other support.